revolv/project/models.py

```diff
@@ -336,6 +330,32 @@ def amount_donated_organically(self):
         )["amount__sum"] or 0.0
 
     @property
+    def location_street(self):
+        """
+        :return: a string of the street name of the location of this project
```

Comment:
I think we should mention in these docstrings that the function will return "" if the location is malformed